tools/DevicesApiTester/Commands/Pwm/PwmCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Pwm;
+using CommandLine;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.Pwm
+{
+    public abstract class PwmCommand : DebuggableCommand
+    {
+        [Option('d', "driver", HelpText = "The PwmDriver to use: { Windows | Unix }", Required = false, Default = PwmDriverType.Windows)]
+        public PwmDriverType Driver { get; set; }
+
+        [Option("pwm-chip", HelpText = "The PWM chip (controller) to use", Required = false, Default = 0)]
+        public int PwmChip { get; set; }
+
+        [Option("pwm-channel", HelpText = "The PWM channel (pin) to use", Required = false, Default = 0)]
+        public int PwmChannel { get; set; }
+
+        [Option('c', "dutycycle", HelpText = "The duty cycle for PWM output from 1-100", Required = false, Default = 50)]
+        public int DutyCycle { get; set; }
+

```

Comment:
Should be a `double`, and the help text should state the range to indicate such: `from 1.0 - 100.0`

Reply:
will change